setup/rhel72-linuxonecc/ansible-playbook.yaml

```diff
@@ -0,0 +1,119 @@
+---
+- hosts: iojs-build-rhel72-linuxonecc
+
+  remote_user: root
+
+  tasks:
+
+    - include_vars: ansible-vars.yaml
+      tags: vars
+
+    - name: General | yum Update
+      yum: name=* state=latest
+      tags: general
+
+    - name: General | Install required packages
+      yum: name={{ item }} state=latest
+      with_items: packages
+      tags: general
+
+    - name: NTP | Run initial NTP
+      command: ntpdate -u pool.ntp.org
+      tags: ntp
+
+    - name: NTP | Enable NTP
+      command: chkconfig ntpd on
+      tags: ntp
+
+    - name: NTP | Start NTP
+      service: name=ntpd state=started enabled=yes
+      tags: ntp
+
+    - name: General | Increase file descriptor limits
+      lineinfile: dest=/etc/security/limits.conf line="{{ item }}"
+      with_items:
+        - "*         hard    nofile      500000"
+        - "*         soft    nofile      500000"
+        - "root      hard    nofile      500000"
+        - "root      soft    nofile      500000"
+      tags: general
+
+    - name: User | Add {{ server_user }} user
+      user: name="{{ server_user }}" shell=/bin/bash
+      tags: user
+
+    - name: User | check if larger disk has file system 
+      command: file -s /dev/dasdb1
+      register: result
+      tags: user
+
+    - name: User | create file system on larger disk
+      command: mkfs.ext4 -L /data /dev/dasdb1
+      when: 'result.stdout == "/dev/dasdb1: data"'
+      tags: user
+
+    - name: User | create mount for larger disk
+      command: mkdir /data creates=/data
+      tags: user
+
+    - name: User | add mount to fstab
+      lineinfile: dest=/etc/fstab
+                  regexp='^'
+                  line='/dev/dasdb1                          /data                   ext4    defaults        0 0'
+                  state=present
```

Comment:
Thanks I guess I still need to learn more of what is available in ansible already.